api/internal/core/store/store_test.go

```diff
@@ -580,7 +580,9 @@ func TestGenericStore_Create(t *testing.T) {
 			createCalled = true
 			assert.Equal(t, tc.wantKey, args[1], tc.caseDesc)
 			input := TestStruct{}
-			_ = json.Unmarshal([]byte(args[2].(string)), &input)
+			if err := json.Unmarshal([]byte(args[2].(string)), &input); err != nil {
+				panic(err)
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/97825678-58136580-1cfa-11eb-8eea-e2e4ba62e722.png)

Not sure if this format is ok 🤔

Reply:
Kindly resolve this conversation if this format if ok.

Reply:
use `assert` instead of `panic`?